internal/server/authn/errors.go

```diff
@@ -0,0 +1,13 @@
+package authn
+
+import (
+	"fmt"
+
+	"github.com/infrahq/infra/internal"
+)
+
+var (
+	ErrInvalidProviderURL          = fmt.Errorf("%w: %s", internal.ErrBadRequest, "invalid provider url")
```

Comment:
I appreciate the consistent formatting but the second format string isn't needed

```suggestion
	ErrInvalidProviderURL          = fmt.Errorf("%w: invalid provider url", internal.ErrBadRequest)
```

Reply:
Made this fix